src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/BatchingWorkQueue.cs

```diff
@@ -120,7 +119,7 @@ private void StartWorker()
             state =>
             {
                 Assumes.NotNull(state);
-                ((BatchingWorkQueue)state).Timer_TickAsync().Forget();
+                _ = ((BatchingWorkQueue)state).Timer_TickAsync();
```

Comment:
FWIW, I moved this to `Forget()` since it's a bit clearer.

Reply:
I guess I'm wondering if you could add this extension method to Utilities.Shared instead of just removing usages.